app/src/pages/playground/PlaygroundChatTemplate.tsx

```diff
@@ -32,22 +55,31 @@ export function PlaygroundChatTemplate(props: PlaygroundChatTemplateProps) {
     throw new Error(`Invalid template type ${template.__type}`);
   }
 
+  const sensors = useSensors(
```

Comment:
this is not necessary but does make the keyboard sorting much much better, can remove if we don't care